### ISSUE TITLE
Role details module - tabs cannot be reachable via the keyboard

### ISSUE THREAD

**Issue by yusufttur** [shopware/platform#8523]
### Shopware Version

6.7.0.0

### Affected area / extension

Extension:B2B-Suite

### Actual behaviour

The tabs in the role details on the company detail page cannot be reachable via the keyboard.
There is no way to perform the function using only the keyboard on the same screen or on a qualifying conforming alternate version.
https://docs.deque.com/issue-help/1.0.0/en/keyboard-inaccessible

https://github.com/user-attachments/assets/dbd8fb39-efb3-4569-8066-7c524f10ed09

### Expected behaviour

The tabs in the role details on the company detail page should be reachable via the keyboard.

### How to reproduce

Try to reach the tabs in the role details on the company detail page via the keyboard.
